Sccp_manager.inc/aminterface.class.php

```diff
@@ -51,7 +54,14 @@ public function __construct($parent_class = null)
         $this->_socket = false;
         $this->_connect_state = false;
         $this->_error = array();
-        $this->_config = array('host' => 'localhost', 'user' => '', 'pass' => '', 'port' => '5038', 'tsoket' => 'tcp://', 'timeout' => 30, 'enabled' => true);
+        $this->_config = array('host' => 'localhost',
+                        'user' => '',
+                        'pass' => '',
+                        'port' => '5038',
+                        'tsoket' => 'tcp://',
```

Comment:
tsoket => socket or socket type ?

Reply:
Russian for socket? or maybe PHP reserved. Will look and change, but is correct